packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useUID} from 'react-uid';
+import {useComboboxPrimitive, UseComboboxPrimitiveProps} from '@twilio-paste/combobox-primitive';
+import {ChevronDownIcon} from '@twilio-paste/icons/esm/ChevronDownIcon';
+import {Box} from '@twilio-paste/box';
+import {FormControlWrapper, FormHelpText, FormLabel, InputElement, FormInputProps} from '@twilio-paste/form';
+import {ComboboxInputWrapper} from './ComboboxInputWrapper';
+import {ComboboxListbox} from './ComboboxListbox';
+import {ComboboxListboxOption} from './ComboboxListboxOption';
+
+const ComboboxIconWrapper: React.FC<{children: React.ReactNode}> = ({children}) => (
+  <Box
+    alignItems="center"
+    display="inline-flex"
+    position="absolute"
+    pointerEvents="none"
+    right="space40"
+    top="50%"
+    transform="translateY(-50%)"
+    zIndex="zIndex10"
+  >
+    {children}
+  </Box>
+);
+
+export interface ComboboxProps extends Omit<FormInputProps, 'id' | 'type' | 'value'> {
+  autocomplete?: boolean;
+  helpText?: string | React.ReactNode;
+  items: UseComboboxPrimitiveProps<any>['items'];
+  labelText: string | NonNullable<React.ReactNode>;
+  onInputValueChange?: UseComboboxPrimitiveProps<any>['onInputValueChange'];
+  itemToString?: UseComboboxPrimitiveProps<any>['itemToString'];
+  optionTemplate?: (item: string | {}) => React.ReactNode;
+}
+
+/* eslint-disable react/no-array-index-key */
+const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
+  (
+    {
+      autocomplete,
+      disabled,
+      hasError,
+      helpText,
+      insertAfter,
+      insertBefore,
+      items,
+      itemToString,
+      labelText,
+      optionTemplate,
+      onInputValueChange,
+      required,
+      ...props
+    },
+    ref
+  ) => {
+    const defaultItemToString = (item: string): string => item;
+    const {
+      getComboboxProps,
+      getInputProps,
+      getItemProps,
+      getLabelProps,
+      getMenuProps,
+      getToggleButtonProps,
+      highlightedIndex,
+      isOpen,
+      selectedItem,
+    } = useComboboxPrimitive({
+      items,
+      onInputValueChange,
+      itemToString: itemToString || defaultItemToString,
+    });
+    const helpTextId = useUID();
+    return (
+      <Box position="relative">
+        <FormLabel disabled={disabled} required={required} {...getLabelProps()}>
+          {labelText}
+        </FormLabel>
+        <FormControlWrapper
+          disabled={disabled}
+          hasError={hasError}
+          insertBefore={insertBefore}
+          insertAfter={insertAfter}
+        >
+          <ComboboxInputWrapper {...getComboboxProps({role: 'combobox'})}>
+            <InputElement
+              {...getInputProps({disabled})}
+              {...getToggleButtonProps({tabIndex: 0})}
+              {...(!autocomplete ? {value: selectedItem || ''} : undefined)}
```

Comment:
Wonder if you can use this syntax on line 71 to save us having to create a default item to string function.

`{...(itemToString ? : itemToString : undefined)}`

Reply:
Looks like that translates to: `itemToString || undefined`, so back to where we were before adding the default item to string function.